Description: detect to avoid trayWindow->window is None
Origin: upstream
Author: csslayer <wengxt@gmail.com>
Last-Update: 2011-10-18
--- a/src/TrayWindow.c
+++ b/src/TrayWindow.c
@@ -127,7 +127,7 @@
         name = "tray_inactive";
 
     LightUIImage* image = LoadImage(lightui, name);
-    if (image)
+    if (image && trayWindow->window != None)
     {
         DrawImage(dpy, trayWindow->window, image, 0, 0, trayWindow->size, trayWindow->size);
     }
